rct229/rules/section23/section23rule8.py

```diff
@@ -120,20 +109,27 @@ def rule_check(self, context, calc_vals=None, data=None):
                     ]
                     for idx in range(11)
```

Comment:
I don't really like the idea of having a "magic number" in the code.
First of all, `SUPPLY_AIRFLOW_COEFF` used this 11 to create a list.
In here `target_validation_points` also used 11 to create a 2-D list.
I think we can create a global variable called `VALIDATION_POINT_LENGTH = 11`  and use it to replace the magic numbers in the code.

To further stretch the logic, in the rule check condition, instead of 
'''python
len(output_validation_points) == len(
                    target_validation_points
                )
'''
as we are certain about the length of the target validation points, we could update this to 
'''python
len(output_validation_points) == VALIDATION_POINTS_LENGTH
'''

Reply:
Great point! Fixed.